app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -171,6 +172,7 @@ func newInstallControlPlaneCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
 	cmd.Flags().StringVar(&args.SdsTlsKey, "sds-tls-key", args.SdsTlsKey, "TLS key for the SDS server")
 	cmd.Flags().StringVar(&args.KdsTlsCert, "kds-tls-cert", args.KdsTlsCert, "TLS certificate for the KDS server")
 	cmd.Flags().StringVar(&args.KdsTlsKey, "kds-tls-key", args.KdsTlsKey, "TLS key for the KDS server")
+	cmd.Flags().StringVar(&args.KdsGlobalAddress, "kds-global-address", args.KdsGlobalAddress, "URL of Global Kuma CP")
```

Comment:
If I remember correctly, Global initializes KDS connection to Remote and everything is exchanged on this connection. Why do we need `KdsGlobalAddress`?

Reply:
Discussed offline that Zone resource will contain only Ingress Address and not Remote CP address. Will this be removed in this PR?

Reply:
No, we discussed with @nickolaev that current PR should leave all configs as they are. Next PR will cleanup configs and ZoneResource